cypress/integration/dashboard.js

```diff
@@ -0,0 +1,143 @@
+            .then(resp => {
+              const customerId = resp.body.data.customerCreate.user.id;
+              const customerEmail = resp.body.data.customerCreate.user.email;
+              shippingUtils
+                .createShipping(
+                  channelId,
+                  randomName,
+                  json.plAddress,
+                  shippingPrice
+                )
+                .then(() => {
+                  const shippingId = shippingUtils.getShippingMethodId();
+                  const warehouseId = shippingUtils.getWarehouseId();
+                  productsUtils
+                    .createTypeAttributeAndCategoryForProduct(randomName)
+                    .then(() => {
+                      const productTypeId = productsUtils.getProductTypeId();
+                      const attributeId = productsUtils.getAttributeId();
+                      const categoryId = productsUtils.getCategoryId();
+                      productsUtils
+                        .createProductInChannel(
+                          randomName,
+                          channelId,
+                          warehouseId,
+                          10,
+                          productTypeId,
+                          attributeId,
+                          categoryId,
+                          productPrice
+                        )
+                        .then(() => {
+                          const variantsList = productsUtils.getCreatedVariants();
+
+                          // Create order ready to fulfill
+                          ordersUtils.createReadyToFulfillOrder(
+                            customerId,
+                            shippingId,
+                            channelId,
+                            variantsList
+                          );
+
+                          // Create order waiting for capture
+                          ordersUtils.createWaitingForCaptureOrder(
+                            channelSlug,
+                            customerEmail,
+                            variantsList,
+                            shippingId
+                          );
+                        });
+
+                      // Create product out of stock
+                      productsUtils.createProductInChannel(
+                        randomNameProductOutOfStock,
+                        channelId,
+                        warehouseId,
+                        0,
+                        productTypeId,
+                        attributeId,
+                        categoryId,
+                        productPrice
+                      );
+                    });
+                });
+            });
+        });
+    });
+    cy.visit("/");
+    cy.get(HEADER_SELECTORS.channelSelect)
+      .click()
+      .get(HEADER_SELECTORS.channelSelectList)
+      .contains(randomName)
+      .click()
+      .waitForGraph("Home");
+    const salesRegexp = new RegExp(`\\D+${sales}[\,.]00\\D*`);
```

Comment:
Sales is exact value. Regex is used because on my local i get "PLN 56.00", on staging, "56,00 zł". I want this test to pass regardless of whether if there will be comma, or dot, "zł" or "PLN".